src/metpy/calc/thermo.py

```diff
@@ -45,7 +45,10 @@ def relative_humidity_from_dewpoint(temperature, dewpoint):
     -------
     `pint.Quantity`
         Relative humidity
+        Notes
+        _____
```

Comment:
This is over-indented. Also, can you add a blank line above the heading?
```suggestion

    Notes
    _____
```